src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -73,36 +73,43 @@ public static HearingDetails buildHearingDetails(HearingWrapper wrapper, Referen
 
     public static boolean shouldBeAutoListed(@Valid SscsCaseData caseData) {
         // TODO Future Work
-        return !isCaseLinked(caseData);
+        return !isCaseLinked(caseData)
+                && !isCaseUrgent(caseData);
+    }
+
+    public static boolean isCaseUrgent(@Valid SscsCaseData caseData) {
+        return isYes(caseData.getUrgentCase());
     }
 
     public static String getHearingType() {
         return SUBSTANTIVE.getHmcReference();
     }
 
     public static HearingWindow buildHearingWindow(@Valid SscsCaseData caseData, boolean autoListed) {
-
-        if (!autoListed || isBlank(caseData.getDwpResponseDate())) {
-            LocalDate dateRangeStart = LocalDate.now().plusDays(DAYS_TO_ADD_HEARING_WINDOW_TODAY);
-            return HearingWindow.builder()
-                    .dateRangeStart(dateRangeStart)
-                    .dateRangeEnd(null)
-                    .build();
-        }
-        LocalDate dwpResponded = LocalDate.parse(caseData.getDwpResponseDate());
-
-        LocalDate dateRangeStart = isYes(caseData.getUrgentCase())
-                ? dwpResponded.plusDays(DAYS_TO_ADD_HEARING_WINDOW_DWP_RESPONDED_URGENT_CASE)
-                : dwpResponded.plusDays(DAYS_TO_ADD_HEARING_WINDOW_DWP_RESPONDED);
-
-
         return HearingWindow.builder()
                 .firstDateTimeMustBe(getFirstDateTimeMustBe())
-                .dateRangeStart(dateRangeStart)
+                .dateRangeStart(getHearingWindowStart(caseData, autoListed))
                 .dateRangeEnd(null)
                 .build();
     }
 
+    public static LocalDate getHearingWindowStart(@Valid SscsCaseData caseData, boolean autoListed) {
+
+        LocalDate startDate = LocalDate.now().plusDays(DAYS_TO_ADD_HEARING_WINDOW_TODAY);
+        if (isBlank(caseData.getDwpResponseDate())) {
+            return startDate;
+        }
+
+        LocalDate dwpResponded = LocalDate.parse(caseData.getDwpResponseDate());
+        if (isCaseUrgent(caseData)) {
+            startDate = dwpResponded.plusDays(DAYS_TO_ADD_HEARING_WINDOW_DWP_RESPONDED_URGENT_CASE);
+        } else if (autoListed) {
+            startDate = dwpResponded.plusDays(DAYS_TO_ADD_HEARING_WINDOW_DWP_RESPONDED);
+        }
```

Comment:
Minor (consider for future work ONLY): This would be more easily read if it were extracted in it's own method